AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/participant-details.model.ts

```diff
@@ -21,9 +21,9 @@ export class ParticipantDetailsModel {
         isInterpretee: boolean
     ) {
         this.ParticipantId = participantId;
-        this.FirstName = firstName;
-        this.LastName = lastName;
-        this.Title = title === undefined ? '' : title;
+        this.FirstName = firstName === null ? '' : firstName;
```

Comment:
Maybe change this to
`!firstName ? '' : firstName`
Then it will handle all cases?

Reply:
Both yes, undefined was not handling the null
As a result an ugly "null" was displayed on the UI